packages/design/src/DataTable/Cells.tsx

```diff
@@ -80,14 +81,39 @@ export const DateCell = ({ data }: { data: Date }) => (
   <Cell>{displayDate(data)}</Cell>
 );
 
+const TextEllipsisLabelContent = ({
+  text,
+  maxLength,
+}: {
+  text: string;
+  maxLength: number;
+}) => {
+  let displayText = text;
+  const isLonger = text.length > maxLength;
+
+  if (isLonger) {
+    displayText = text.substring(0, 100);
+  }
+
+  return (
+    <LabelContent title={text} dots={isLonger}>
+      {displayText}
+    </LabelContent>
+  );
+};
```

Comment:
not necessary (minus the `title` feature, we probably need that), see my other comment about the CSS ellipsis feature

Reply:
I have answered your previous comments about why actually using CSS ellipsis is not a good move here in my opinion, and why I made the decision not to do that. 

``LabelConent`` component is actually quite needed because CSS ellipsis does not simplify the code and styling for tags.